operator-common/src/main/java/io/strimzi/operator/common/config/ConfigParameterParser.java

```diff
@@ -2,13 +2,14 @@
  * Copyright Strimzi authors.
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
-package io.strimzi.operator.common.operator.resource;
+package io.strimzi.operator.common.config;
 
 
 import io.fabric8.kubernetes.api.model.LocalObjectReference;
 import io.fabric8.kubernetes.api.model.LocalObjectReferenceBuilder;
 import io.strimzi.operator.common.InvalidConfigurationException;
 import io.strimzi.operator.common.model.Labels;
+import io.strimzi.operator.common.operator.resource.AbstractNamespacedResourceOperator;
```

Comment:
I was wondering about what you had in the description ...

> ConfigParameter and ConfigParameterParser are used for configuration of our operators and have nothing to do with the Kubernetes Resource Operators.

But then we have a reference to something related to resource operator because of the `ANY_NAMESPACE` constant. Maybe such constant belongs to this class now?

Reply:
Good point. I did not notice that. I will decouple it as this coupling makes no sense.

Reply:
I moved it to the `ConfigParameter` class. I also made use of it in other classes which so far used a separate `"*"` String.